### PR TITLE
Feature fe discovery page #39

### PR DESCRIPTION
**Pull request by Rhut-virani**
-- added discovery page

### CODE REVIEW

File: client/src/themes/theme.js [hunk 1]
```diff
@@ -3,13 +3,13 @@ import { createMuiTheme } from '@material-ui/core';
...
   typography: {
     fontFamily: 'Poppins , sans-serif',
-    fontSize: 12,
-    h6: {
-      fontSize: 16,
-    },
-    button: {
-      fontSize: 9,
-    },
+    // fontSize: 12,
```
Comment (bonnieli) [inline client/src/themes/theme.js]: are these not needed anymore?

File: server/controllers/contests.js [hunk 2]
```diff
@@ -5,113 +5,113 @@ const { userCheck } = require('../helpers/userCheck');
...
-    if (!contests) {
-      return res.status(404).json({ msg: 'There are no contests' });
-    }
-    res.status(200).json(contests);
-  } catch (error) {
-    console.error(error.message);
-    res.status(500).json({ msg: 'Server error - 500' });
-  }
+		if (!contests) {
+			return res.status(404).json({ msg: "There are no contests" });
```
Comment (bonnieli) [inline server/controllers/contests.js:56]: looks like your prettier is linting some files - lets make sure you and @UVStudio 's linters are the same configs

File: client/src/themes/theme.js (hunk 1 above)
Comment (Rhut-virani) [reply client/src/themes/theme.js]: removed these lines

Comment (UVStudio) [reply client/src/themes/theme.js]: Yes removing them is fine. Rhut and I have agreed that he has final say over fontsize.

**Pull request merged**